dashboard/src/actions/tocActions.js

```diff
@@ -41,6 +45,20 @@ export const fetchTOC =
     dispatch({ type: TYPES.COMPLETED });
   };
 
+const setOptions = (data, isParent, keyPath, isDirectory) => {
+  const options = data.map((item) => ({
+    name: item.name,
+    id: isParent ? `${keyPath}*${item.name}` : item.name,
+    isDirectory,
```

Comment:
It doesn't complain because that's a Javascript feature: a single value in a JSON object represents a field with the variable name as key and the variable's value. It's preferred when the variable name matches the desired key...

Reply:
Ah.  Well alright then.  (👃🤏)